main.js

```diff
@@ -0,0 +1,21 @@
+// ADDS FEATURED CLASS TO SECTION ELEMENT
+
+const sectionElement = document.querySelector("section");
+sectionElement.classList.add("featured")
+
+
+// CREATES AN ARTICLE ELEMENT
+
+const articleElement = document.querySelector("article");
+// articleElement.createElement("")
```

Comment:
A lot of important DOM manipulation methods are stored on the "document" object. E.g. if you want to create a new article element, you could call `document.createElement("article")`